src/main/java/seedu/address/model/task/Task.java

```diff
@@ -96,13 +109,14 @@ public boolean equals(Object other) {
 
         Task otherTask = (Task) other;
         return taskName.equals(otherTask.taskName)
-                && taskDescription.equals(otherTask.taskDescription);
+                && taskDescription.equals(otherTask.taskDescription)
+                && priority.equals(otherTask.priority);
```

Comment:
In the event 2 tasks have the same name, description and priority but have different isDone and date data fields, should they be considered equal to each other?